readthedocs/donate/templates/donate/promo_404.html

```diff
@@ -11,6 +11,7 @@
 {% endblock %}
 
 {% block content %}
+<div style="margin-left: auto; margin-right: auto; width: 900px; text-align: center">
```

Comment:
That seems to go against the goal of keeping the donate stuff as it's own app? I think it probably makes sense for a small use-case like this.